perun-core/src/main/java/cz/metacentrum/perun/core/impl/AttributesManagerImpl.java

```diff
@@ -4508,21 +4509,52 @@ public Object getAttributesModule(PerunSession sess, AttributeDefinition attribu
 	}
 
 	@Override
-	public void initAttributeModules(ServiceLoader<AttributesModuleImplApi> modules) {
-		for (AttributesModuleImplApi module : modules) {
-			attributesModulesMap.put(module.getClass().getName(), module);
-			log.debug("Module {} loaded.", module.getClass().getSimpleName());
+	public AttributesModuleImplApi getUninitiatedAttributesModule(PerunSession sess, AttributeDefinition attributeDefinition) {
```

Comment:
There is a mixing of ``uninitialized`` and ``uninitiated``, first should be correct.

Reply:
Fixed.